scripts/train/benchmarking/submit_benchmarks.py

```diff
@@ -351,30 +406,43 @@ def get_integrations(project: str,
 def run_config(config: Tuple[str, int, int, str, str, int, str],
                args: argparse.Namespace):
     model_yaml, max_seq_len, global_train_batch_size, cluster, gpu_type, gpu_num, precision = config
-
-    integrations = get_integrations(
-        args.project,
-        git_branch=args.git_branch,
-        git_commit=args.git_commit,
-        wandb=args.wandb)  # point to git repo and potentially wandb
-
-    # Define our command
-    if args.data_remote is not None:
-        command = """
-        cd examples/scripts
-
-        composer train/train.py /mnt/config/parameters.yaml
+    integrations = [{
+        'integration_type': 'git_repo',
+        'git_repo': 'crinard/llm-foundry',
+        'git_branch': 'add_attns',
+        'pip_install': '-e .[gpu]',
+    }, {
+        'integration_type': 'wandb',
+        'entity': 'mosaic-ml',
+        'project': args.project
+    }]
+
+    command = ''
```

Comment:
I think all of this stuff is no longer necessary?

Reply:
this comment was in reference to the nightly installs
now removed.